src/components/Chart/CrosshairTooltip/index.jsx

```diff
@@ -1,17 +1,22 @@
 import dayjs from 'dayjs';
 import PropTypes from 'prop-types';
+import { useMedia } from 'react-use';
 
 import { Typography } from '../../../Theme';
-import { formatChartValueByType } from '../../../utils';
+import { formatChartValueByType, getWeekFormattedDate } from '../../../utils';
 import { Wrapper } from './styled';
 
-const CrosshairTooltip = ({ title, active, payload, dataType }) => {
+const CrosshairTooltip = ({ title, active, isWeeklyActive, payload, dataType }) => {
+  const below500 = useMedia('(max-width: 500px)');
```

Comment:
why `500px`?

We can also improve the naming

```suggestion
  const isMobile = useMedia('(max-width: 500px)');
```
this is just a naming example that it's more clear.

Reply:
I've used `500px` because below that the date range + the % change can't fit and that causes the second date to go below the first one.

I'll rename it to `isBelow500`.

Reply:
Maybe add px at the end? 

`isBelow500px` is more explicit, without that I have to read the useMedia code.